src/main/java/it/pn/frontend/e2e/common/NotificheDestinatarioPage.java

```diff
@@ -15,6 +15,8 @@ public class NotificheDestinatarioPage extends BasePage{
 
     @FindBy(id = "iunMatch")
     WebElement codiceIunTextField;
+    @FindBy(xpath = "//button[contains(text(),'Ricevuta di consegna PEC')]")
```

Comment:
delete, is just inserted by me in my branch

Reply:
merged?

Reply:
not, but is in my pr